content/docs/command-reference/push.md

```diff
@@ -193,16 +192,18 @@ Finally, we used `dvc status` to double check that all data had been uploaded.
 
 https://www.youtube.com/watch?v=FYmmiAz81G4
 
-Let's take a detailed look at what happens to the
-[cache directory](/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory)
-as you run an experiment locally and push data to remote storage. To set the
-example consider having created a <abbr>workspace</abbr> that contains some code
-and data, and having set up a remote.
+Let's take a detailed look at what happens to the [cache directory] as you run
+an experiment locally and push data to remote storage. To set the example
+consider having created a <abbr>project</abbr> with some code, data, and a
+`dvc remote` setup.
 
 Some work has been performed in the workspace, and new data is ready for
-uploading to the [remote](/doc/command-reference/remote). `dvc status --cloud`
-will list several files in `new` state. We can see exactly what that means by
-looking in the project's <abbr>cache</abbr>:
+uploading to the remote. `dvc status --cloud` will list several files in `new`
+state. We can see exactly what that means by looking in the project's
+<abbr>cache</abbr>:
+
+[cache directory]:
```

Comment:
Not clear if we have any consistency or guideline to where the "shared" link definitions are (sometime towards the top, sometimes mixed, sometimes bottom, sometimes it's spread)

Reply:
I try to put them at the bottom of each uninterrupted chunk of text.